float64[] angle_vector
---
string response